do_everything.sh

```diff
@@ -264,6 +281,28 @@ if [[ $skip -ne 1 ]] ; then
 
 fi
 
+# Before build
```

Comment:
Can you add a message to the user, so we know what the script is doing?

Something like:
```
echo
echo -e '\e[34mBuilding ....\e[39m'
echo
```